src/apis/computed.ts

```diff
@@ -9,7 +9,7 @@ interface Option<T> {
 }
 
 // read-only
-export function computed<T>(getter: Option<T>['get']): Ref<T>;
+export function computed<T>(getter: Option<T>['get']): Readonly<Ref<T>>;
```

Comment:
I think this is more correct

```suggestion
export function computed<T>(getter: Option<T>['get']): Readonly<Ref<Readonly<T>>>;
```